src/Kernels/DynamicRupture.cpp

```diff
@@ -313,20 +267,6 @@ void seissol::kernels::DynamicRupture::flopsGodunovState( DRFaceInformation cons
   o_nonZeroFlops += dynamicRupture::kernel::evaluateAndRotateQAtInterpolationPoints::nonZeroFlops(faceInfo.minusSide, faceInfo.faceRelation);
   o_hardwareFlops += dynamicRupture::kernel::evaluateAndRotateQAtInterpolationPoints::hardwareFlops(faceInfo.minusSide, faceInfo.faceRelation);
 
-  o_nonZeroFlops += dynamicRupture::kernel::computeSlipRateInterpolated::NonZeroFlops;
-  o_hardwareFlops += dynamicRupture::kernel::computeSlipRateInterpolated::HardwareFlops;
-
-  o_nonZeroFlops += dynamicRupture::kernel::computeTractionInterpolated::NonZeroFlops;
-  o_hardwareFlops += dynamicRupture::kernel::computeTractionInterpolated::HardwareFlops;
-
-  o_nonZeroFlops += dynamicRupture::kernel::computeSquaredNormSlipRateInterpolated::NonZeroFlops;
-  o_hardwareFlops += dynamicRupture::kernel::computeSquaredNormSlipRateInterpolated::HardwareFlops;
-  o_nonZeroFlops += 2*tensor::squaredNormSlipRateInterpolated::size();
-  o_hardwareFlops += 2*tensor::squaredNormSlipRateInterpolated::size();
-
-  o_nonZeroFlops += dynamicRupture::kernel::accumulateFrictionalEnergy::NonZeroFlops;
-  o_hardwareFlops += dynamicRupture::kernel::accumulateFrictionalEnergy::HardwareFlops;
-  
   o_nonZeroFlops *= CONVERGENCE_ORDER;
```

Comment:
remove (+ line below)?

Reply:
Why? We still have a loop over CONVERGENCE_ORDER

https://github.com/SeisSol/SeisSol/blob/7ae55cfa49d5a9cd6312d48a10a58bcb61a0d3ca/src/Kernels/DynamicRupture.cpp#L143-L145

Reply:
ok, sorry